packages/plugins/response-cache/src/plugin.ts

```diff
@@ -310,9 +310,14 @@ export function useResponseCache<PluginContext extends Record<string, any> = {}>
                 scopePerSchemaCoordinate[`${type.name}`] = cacheControl.scope;
               }
             });
-            return type;
-          },
-          [MapperKind.FIELD]: (fieldConfig, fieldName, typeName) => {
+          }
+          return type;
+        },
+        [MapperKind.FIELD]: (fieldConfig, fieldName, typeName) => {
+          const id = `${typeName}.${fieldName}`;
```

Comment:
Perhaps call this variable `schemaCoordinates` ?
```suggestion
          const schemaCoordinates = `${typeName}.${fieldName}`;
```